ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -353,6 +353,18 @@ objects:
         sshKey: |
           ${SSH_PUB_KEY}
         EOF
+            export OVA_URL="$(jq -r '.baseURI + .images["vmware"].path' /var/lib/openshift-install/rhcos.json)"
+            echo "Creating a template for the VMs from ${OVA_URL}..."
+            mkdir /tmp/rhcos
+            curl -L -o /tmp/rhcos/rhcos.ova "${OVA_URL}"
```

Comment:
Lets hash `OVA_URL` and use it as a template name to avoid additional uploads?

Reply:
That is my eventual goal. How would we clean up old unused templates?

Reply:
Can't think of any automated method except of manual pruning every N days, as 4.x branch would use different RHCOS images

Reply:
Sounds good. My initial concern was that there would be a lot of unique OVAs. But it looks like we will not have OVA changes too frequently.

ae387060...2a3b5135

Reply:
And 2a3b5135...b0868d37